<?xml version="1.0" ?>

<launch> 

<group ns="turtlebot3_burger">
<!-- -->


		<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
				
		<rosparam file="$(find mirzaagha_rl)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
		<rosparam file="$(find mirzaagha_rl)/param/local_costmap_params.yaml" command="load" />
		<rosparam file="$(find mirzaagha_rl)/param/global_costmap_params.yaml" command="load" />
		<rosparam file="$(find mirzaagha_rl)/param/dwa_local_planner_params.yaml" command="load" />
		<rosparam file="$(find mirzaagha_rl)/param/move_base_params.yaml" command="load" />
		
		
		</node>
	</group>
</launch>
